src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryFunction.cs

```diff
@@ -5,24 +5,39 @@
 using NServiceBus;
 using SFA.DAS.ApprenticeFeedback.Jobs.Domain.Messages.Commands;
 using SFA.DAS.ApprenticeFeedback.Jobs.Infrastructure;
+using System;
 
 namespace SFA.DAS.ApprenticeFeedback.Jobs.Functions
 {
     public class ApprenticeFeedbackSummaryFunction
     {
+        private readonly ILogger<GenerateFeedbackTransactionsFunction> _log;
         private readonly IFunctionEndpoint _endpoint;
 
-        public ApprenticeFeedbackSummaryFunction(IFunctionEndpoint endpoint) => _endpoint = endpoint;
+        public ApprenticeFeedbackSummaryFunction(ILogger<GenerateFeedbackTransactionsFunction> log, IFunctionEndpoint endpoint)
+        {
+            _log = log;
+            _endpoint = endpoint;
+        }
 
         [FunctionName("ApprenticeFeedbackSummaryTimer")]
         public void ApprenticeFeedbackSummaryTimer(
             [TimerTrigger("%FunctionsOptions:ApprenticeFeedbackSummarySchedule%")] TimerInfo timer, 
             ExecutionContext executionContext, 
             ILogger logger)
```

Comment:
Does the `ILogger logger` param need to be here could `_log` variable be used instead?

Reply:
Removed that

Reply:
Fixed